SwiftSyft/Classes/SyftWebSocket.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SyftWebSocket.swift
+//  SwiftSyft
+//
+//  Created by Sasha Bataieva on 25/01/2020.
+//
+
+import Foundation
+import Starscream
+
+public enum WSEventType {
+    case connected
+    case disconnected
+    case text(String)
+    case binary(Data)
+    case error(Error?)
+    case cancelled
+}
+
+public class SyftWebSocket: SocketClientProtocol {
+    weak var delegate: SocketClientDelegate?
```

Comment:
This should be moved to `SyftWebSocketClientiOS12`. It should be public since the one to conform to this will be the signalling client, not the web socket client.

Reply:
fixed